cypress/support/commands.js

```diff
@@ -24,3 +24,21 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 import 'cypress-file-upload'
+Cypress.Commands.add('loginThroughBackend', () => {
```

Comment:
Not the topic of this lecture. Login through the UI is just fine for now :)

Reply:
yeah...I tried but gave up due to user in local storage not being the same as user object in login response